slang-checks/src/test/resources/com/sonarsource/slang/checks/DuplicateBranch.slang

```diff
@@ -0,0 +1,75 @@
+  if (x) {
+    foo;
+    foo;
+  }
+
+  if (x) {
+    foo;
+    foo;
+  } else {
+    bar;
+    bar;
+  }
+
+  if (x) { // handled by S3923
+    foo;
+    foo;
+  } else {
+    foo;
+    foo;
+  }
+
+  if (x) {
+    foo;
+    foo;
+  } else if (y) { // Noncompliant {{This branch's code block is the same as the block for the branch on line 22.}}
```

Comment:
Could you please add a location check on one of those noncompliant example?

Reply:
Done in 2e5ddad.